public/playpass-content.schema.json

```diff
@@ -1,15 +1,16 @@
 {
   "type": "object",
   "properties": {
-    "src": {
-      "type": "string"
+    "songLink": {
+      "type": "string",
+      "description": "Link to your song. Your song links should be based off of the album versions of songs.",
+      "format": "uri",
+      "regex": "^((?:https?:)?\\/\\/)?((?:www|m)\\.)?((?:youtube\\.com|youtu.be))(\\/(?:[\\w\\-]+\\?v=|embed\\/|v\\/)?)([\\w\\-]+)(\\S+)?$"
     },
-    "type": {
```

Comment:
if we're getting rid of the explicit type, we need to add in the audio-ext-element detection on what source type to use based on url parsing

Reply:
oh I see, you're just using pattern matching in the schema and forcing only youtube support for now